packages/playground/src/components/FeatureDisabledScreen.tsx

```diff
@@ -18,6 +22,14 @@ export function FeatureDisabledScreen(props: {
         <div className="flex flex-col items-center gap-1">
           <h2 className="text-lg font-semibold text-foreground">{title}</h2>
           <p className="text-foreground/70 text-center max-w-md">{message}</p>
+          <Button
```

Comment:
Why not make this a link?

Reply:
good point. original idea was to use a "go back" button, which i've now implemented